src/Routing/SitemapLoader.php

```diff
@@ -38,7 +49,7 @@ public function load($resource, $type = null): RouteCollection
 
         $routes = new RouteCollection();
 
-        $providers = $this->container->get('sylius.sitemap_builder')->getProviders();
+        $providers = $this->sitemapBuilder->getProviders();
```

Comment:
While we are improving things anyway, I would opt to move this to combine it at once in line 54. No really need to put it in a separate variable anyway.